elephant/spade.py

```diff
@@ -492,18 +522,22 @@ def concepts_mining(data, binsize, winlen, min_spikes=2, min_occ=2,
         return mining_results, rel_matrix
 
 
-def _build_context(binary_matrix, winlen):
+def _build_context(binary_matrix, winlen, only_windows_with_first_spike=True):
     """
     Building the context given a matrix (number of trains x number of bins) of
     binned spike trains
     Parameters
     ----------
     binary_matrix : numpy.array
-        Binary matrix containing the binned spike trais
+        Binary matrix containing the binned spike trains
     winlen : int
         Length of the binsize used to bin the data
+    only_windows_with_first_spike : bool
+        Whether to consider every window or only the one with a spike in the
```

Comment:
I think some motivation for the parameter is needed, like why one should consider it to be True.